### PR TITLE
Comment component

### PR DESCRIPTION
**Pull request by taleymirza**
#97

### CODE REVIEW

**Comment by taleymirza** [comment]
@mashhoodr it's blocked due to backend issue in API.

**Comment by mashhoodr** [comment]
@taleymirza lets mock the response and wrap this up today

**Comment by taleymirza** [comment]
It's already getting comments from backend but unable to add new comments due to API backend issue

File: client/src/components/Comments/CommentList.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react';
+import { Col, Row } from 'reactstrap';
+import moment from 'moment';
+
+class CommentList extends Component {
+  render() {
+    return (
+      <ul className="commentList">
+        {this.props.comments.map(comment => (
```
Comment (oahmedqureshi) [inline client/src/components/Comments/CommentList.js]: Please break this down into sub component e.g create new component "CommentListItem" pass comment as prop to it then map "<CommentListItem comment={comment}/>"

File: client/src/components/Comments/CommentsBlock.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react';
```
Comment (oahmedqureshi) [inline client/src/components/Comments/CommentsBlock.js]: Clean and readable code good work (Y)

Comment (taleymirza) [reply client/src/components/Comments/CommentsBlock.js]: thanks

File: client/src/components/Comments/CommentList.js (hunk 1 above)
Comment (taleymirza) [reply client/src/components/Comments/CommentList.js]: done!

**Pull request merged**